woocommerce/woocommerce-android
# [Manual Taxes M3] Add Tracking events

### Do not merge until it's targeting trunk

Closes: #9793 


### Description


This PR add the tracking events for Milestone 3 of the Manual Taxes project. 

### Testing instructions


- Go to the Orders Tab
- Create a new order
- add a product
- click on Set New Tax Rate
- toggle the option to `Add this rate to all created orders`
- select a tax  rate
- make sure the event below is tracked with the property auto_tax_rate_enabled":true 
- untoggle the button and make sure the property is set to false

🔵 Tracked: tax_rate_selector_tax_rate_tapped, Properties: {"auto_tax_rate_enabled":true,"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to enable `Add this rate to all created orders`
- back to Order creation screen, click on `Edit Tax Rates Settings`
- make sure the event below is tracked when the bottom sheet is displayed

🔵 Tracked: tax_rate_auto_tax_bottom_sheet_displayed, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to display the bottom sheet (edit tax rate)
- Click on `Set a new rate for this order`
- Make sure the event below is tracked

🔵 Tracked: tax_rate_auto_tax_rate_set_new_rate_for_order_tapped, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}

- Repeat the steps above to display the bottom sheet
- tap on `Clear Address and stop using this rate`
- make sure the event below is tracked

🔵 Tracked: tax_rate_auto_tax_rate_clear_address_tapped, Properties: {"blog_id":221646855,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true,"site_url":"https:\/\/testwoomobile.wpcomstaging.com"}
